huxley/www/js/entry.js

```diff
@@ -20,6 +20,7 @@ var AdvisorProfileView = require('components/AdvisorProfileView');
 var AdvisorRosterView = require('components/AdvisorRosterView');
 var ChairAttendanceView = require('components/ChairAttendanceView');
 var ChairSummaryView = require('components/ChairSummaryView');
+var DelegateCommitteeFeedbackView = require('components/DelegateCommitteeFeedbackView');
```

Comment:
Don't reference the view until it exists

Reply:
Oh whoops I didn't mean to commit that :/ darn.

Well I've been hacking at the frontend a lot in the last few days and should be done with that relatively soon anyways so I'll push those soon!

Reply:
I can push this as is without these; otherwise you'll have to wait until the frontend exists for the PR to close which will mean dealing with new migrations (there's some code I'm going to push up very soon)

Reply:
Yea works for me

Reply:
Can you do that in github or do you need me to push it up?

Reply:
I can do it in Github this time, but generally it's better if you do it on your side